src/components/GuidedSetup/index.js

```diff
@@ -88,6 +92,16 @@ function GuidedSetup({ children }) {
     }
   }, [identityData]);
 
+  useEffect(() => {
+    if (!isIdentityEmpty) {
```

Comment:
Why use this as useEffect instead of just putting it next to where setIsIdentityEmpty is used?